apps/src/templates/manageStudents/ManageStudentsAgeCell.jsx

```diff
@@ -31,7 +31,7 @@ class ManageStudentAgeCell extends Component {
         {!this.props.isEditing && <div>{age}</div>}
         {this.props.isEditing && (
           <select
-            style={{width: 50}}
+            style={{width: 50, marginBottom: 0}}
```

Comment:
Overrides the marginBottom 10 that was bumping this above the center of the cell